### PR TITLE
[Docs] Separate command and result

### PR DESCRIPTION
**Pull request by JStickler**
Now that we've implemented a copy-paste feature for code blocks on the docs website, put commands and the results in separate code blocks in the docs for ease of use.

### CODE REVIEW

File: docs/sources/send-data/promtail/cloud/eks/_index.md [hunk 1]
```diff
@@ -49,14 +52,22 @@ To ship all your pods logs we're going to set up [Promtail]({{< relref "../../..
 
 What's nice about Promtail is that it uses the same [service discovery as Prometheus][prometheus conf], you should make sure the `scrape_configs` of Promtail matches the Prometheus one. Not only this is simpler to configure, but this also means Metrics and Logs will have the same metadata (labels) attached by the Prometheus service discovery. When querying Grafana you will be able to correlate metrics and logs very quickly, you can read more about this on our [blogpost][correlate].
 
-Let's add the Loki repository and list all available charts.
+Let's add the Loki repository and list all available charts. To add the repo, the following command:
```
Comment (ashwanthgoli) [inline docs/sources/send-data/promtail/cloud/eks/_index.md:55]:
```suggestion
Let's add the Loki repository and list all available charts. To add the repo, run the following command:
```

File: docs/sources/operations/loki-canary/_index.md [hunk 2]
```diff
@@ -259,16 +259,23 @@ spec:
...
-# build the binary
-$ make loki-canary
+    ```bash
+    $ git clone https://github.com/grafana/loki
+    ```
 
-# (optionally build the container image)
-$ make loki-canary-image
-```
+1. Build the binary.
```
Comment (ashwanthgoli) [inline docs/sources/operations/loki-canary/_index.md:268]: I don't think the numbering really matters once it renders
just an fyi, ignore if this is intentional :)

File: docs/sources/send-data/promtail/cloud/eks/_index.md [hunk 3]
```diff
@@ -104,8 +125,11 @@ Verify the application is working by running these commands:
 Verify that Promtail pods are running. You should see only two since we're running a two nodes cluster.
 
 ```bash
-kubectl get -n monitoring pods
+kubectl get -n monitoring
```
Comment (ashwanthgoli) [inline docs/sources/send-data/promtail/cloud/eks/_index.md:128]:
```suggestion
kubectl get -n monitoring pods
```

File: docs/sources/send-data/promtail/cloud/eks/_index.md (hunk 1 above)
Comment (JStickler) [reply docs/sources/send-data/promtail/cloud/eks/_index.md:55]: Whoops, good catch!

File: docs/sources/operations/loki-canary/_index.md (hunk 2 above)
Comment (JStickler) [reply docs/sources/operations/loki-canary/_index.md:268]: It's actually a best practice to do the numbering this way.

https://grafana.com/docs/writers-toolkit/write/markdown-guide/#numbered-lists

File: docs/sources/operations/troubleshooting.md [hunk 4]
```diff
@@ -99,8 +99,10 @@ port (`9080` or `3101` if using Helm) locally:
 
 ```bash
 $ kubectl port-forward loki-promtail-jrfg7 9080
-# Then, in a web browser, visit http://localhost:9080/service-discovery
-```
+``````
```
Comment (chaudum) [inline docs/sources/operations/troubleshooting.md:102]: I think there should only be three back-ticks.

Comment (JStickler) [reply docs/sources/operations/troubleshooting.md]: Good catch, thanks!

File: docs/sources/send-data/promtail/cloud/eks/_index.md (hunk 1 above)
Comment (ashwanthgoli) [reply docs/sources/send-data/promtail/cloud/eks/_index.md:55]: i think the latest commit reverted the fix?

File: docs/sources/send-data/promtail/cloud/eks/_index.md (hunk 3 above)
Comment (ashwanthgoli) [reply docs/sources/send-data/promtail/cloud/eks/_index.md:128]: i think the latest commit reverted the fix?

**Pull request merged**